packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -16,8 +17,14 @@
                     data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_information_description' | translate }}"
                     data-description="{{:: $ctrl.tenant.description }}"
                 >
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.goToRenameService()">
+                    <oui-action-menu
+                        aria-label="Actions"
```

Comment:
**nitpick:** Might be great if we can move this to a translation file.

Reply:
This is done